File: Python/Niveau 2/1 - Nombre à virgules et autres outils/Partie B/2 - Augmentation des taxes.py
taxe_actuelle = float(input())
nouvelle_taxe = float(input())
prix_actuel = float(input())

prix_hors_taxe = prix_actuel/(1+(taxe_actuelle/100))

prix_avec_nouvelle_taxe = prix_hors_taxe*(1+(nouvelle_taxe/100))

prix_avec_nouvelle_taxe = round(prix_avec_nouvelle_taxe,2)

print(prix_avec_nouvelle_taxe)

File: C++/Niveau 2/1 - Nombres à virgules et autres outils/Partie B/2 - Augmentation des taxes.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
    double taxeActuelle, nouvelleTaxe, prixActuel;

    cin >> taxeActuelle >> nouvelleTaxe >> prixActuel;

    double prixHorsTaxe = prixActuel/ ( 1 + (taxeActuelle/100));

    double futurPrix = prixHorsTaxe * (1 + (nouvelleTaxe/100));

    cout << round(futurPrix*100)/100 << endl;
}